playwright-e2e/pages/osteo/home/home-page.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, Locator, Page } from '@playwright/test';
+import { OsteoPage } from 'pages/osteo/osteo-page';
+import { HomePageInterface } from 'pages/page-interface';
+
+/**
+ * Landing page, unauthenticated.
+ */
+export default class HomePage extends OsteoPage implements HomePageInterface {
+  constructor(page: Page) {
+    super(page);
+  }
+
+  async waitForReady(): Promise<void> {
+    await expect(this.getLogInButton()).toBeVisible();
+    await expect(this.getJoinCountMeInButton()).toBeVisible();
+  }
+
+  getLogInButton(): Locator {
+    return this.page.locator('.header button[data-ddp-test="signInButton"]:has-text("Log In") >> nth=0');
```

Comment:
I have to use 'nth=0' because the locator finds 2 elements. Same for line 23.
And it is weird, actually.

See the screenshot below
![err](https://user-images.githubusercontent.com/7396837/204634940-cdaa3e37-d7b4-42e3-8ffe-fa8c2daf0250.png)

Reply:
`'.header button[data-ddp-test="signInButton"]` resolves to one element.